models/DataObject/SelectOptions/Config.php

```diff
@@ -104,6 +108,21 @@ public function setGroup(?string $group): static
         return $this;
     }
 
+    public function getAdminOnly(): bool
+    {
+        return $this->adminOnly;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setAdminOnly(?bool $adminOnly): static
+    {
+        $this->adminOnly = $adminOnly ?? false;
```

Comment:
It is not necessary to make the setter nullable, is it?

```suggestion
    public function setAdminOnly(bool $adminOnly): static
    {
        $this->adminOnly = $adminOnly;
```